test/deploy/kubernetes/runtime-pods/components_server_node2.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     metadata:
       labels:
         app: server-node2
+        cortx.io/machine-id: eee340f79047df9bb52fa460615c32a5
```

Comment:
Can we pull this value directly from pod metadata id? Instead of explicitly specifying it